### ISSUE TITLE
CouchDB "last_seq" number changes when adding since=now

### ISSUE THREAD

**Issue by MartinWFL** [apache/couchdb#4549]
## Description

I am getting inconsistent results from CouchDB version 3.2.1.

If I get a list of changes in a database by calling `http://docdb.mycouchdb.com/my_database/_changes` I get a long list of changes in "results" with the final "seq" value matching what it also shows in the "last_seq" property:

```
{"results":[
{"seq":"1-g1AAAABPeJzLYWBgYMxgTmEQTM4vTc5ISXIwNDLXMwBCwxygFFMiQx4Lw38gyMpgTmTMBQqxGxiaWRilJmHTkgUAFM4Udw","id":"_design/timestampsviews","changes":[{"rev":"1-72c0e9b448985fd65beb3cc3a580c588"}]},
.
etc.
.
{"seq":"330-g1AAAABSeJzLYWBgYMxgTmEQTM4vTc5ISXIwNDLXMwBCwxygFFMiQx4Lw38gyMpgTgIq9coFirIbGJpZGKUmYdOVBQBEcxTC","id":"fa3441f98a6643810cd8925d6491ef9f","changes":[{"rev":"1-88db0df4d97e2004c00c87c030bda7b5"}]}
],
"last_seq":"330-g1AAAABSeJzLYWBgYMxgTmEQTM4vTc5ISXIwNDLXMwBCwxygFFMiQx4Lw38gyMpgTgIq9coFirIbGJpZGKUmYdOVBQBEcxTC","pending":0}
```

However if I do `http://docdb.mycouchdb.com/my_database/_changes?since=now` (ie. add "?since=now") because I only want to get "last_seq" I get a completely different value which did not appear in the previous list:

```
{"results":[
],
"last_seq":"333-g1AAAABSeJzLYWBgYMxgTmEQTM4vTc5ISXIwNDLXMwBCwxygFFMiQx4Lw38gyMpgTgIq9c0FirIbGJpZGKUmYdOVBQBE2RTF","pending":0}
```

## Steps to Reproduce

It doesn't do it for all of my databases, nor can I see anything about this particular database that is different than my other databases, so this is difficult to reproduce :disappointed: 

## Expected Behaviour

I  would expect one of two things.

Either `http://docdb.mycouchdb.com/my_database/_changes` should have a couple more entries in "results" and end with "last_seq":"333-g1AAAABSeJzLYWBgYMxgTmEQTM4vTc5ISXIwNDLXMwBCwxygFFMiQx4Lw38gyMpgTgIq9c0FirIbGJpZGKUmYdOVBQBE2RTF"

or `http://docdb.mycouchdb.com/my_database/_changes?since=now` should return the "last_seq" property as "330-g1AAAABSeJzLYWBgYMxgTmEQTM4vTc5ISXIwNDLXMwBCwxygFFMiQx4Lw38gyMpgTgIq9coFirIbGJpZGKUmYdOVBQBEcxTC"

In other words, consistency ... so that "last_seq" is the same regardless of how you called it

## Your Environment

CouchDB 3.2.1 running on AWS

* CouchDB version used: 3.2.1
* Browser name and version: ALL
* Operating system and version: ALL

**Comment by janl** [comment]
This is expected behaviour: https://lists.apache.org/thread/z79yxo9bmxwy4xdpfybx5vhmt47z12tl
